backend/src/services/ApplicationService.js

```diff
@@ -44,7 +79,7 @@ export default class ApplicationService extends BasePocketService {
   async __persistApplicationIfNotExists(application) {
     if (!await this.applicationExists(application)) {
       /** @type {{insertedId: string, result: {n:number, ok: number}}} */
-      const result = await this.persistenceService.saveEntity(APPLICATION_COLLECTION_NAME, application);
+      const result = await this.persistenceService.saveEntity(APPLICATION_COLLECTION_NAME, await this.__encryptApplicationFields(application));
```

Comment:
Can we deconstruct this line, it will be clear if we create a constant for the `await this.__encryptApplicationFields(application)` and then use it in the result line